src/components/calcite-panel/calcite-panel.tsx

```diff
@@ -18,7 +18,7 @@ const SLOTS = {
  * @slot header-content - A slot for adding content in the center of the header.
  * @slot header-leading-content - A slot for adding a `calcite-action` on the leading side of the header.
  * @slot header-trailing-content - A slot for adding a `calcite-action` on the trailing side of the header.
- * @slot footer - A slot for adding `calcite-actions` to the footer.
+ * @slot footer - A slot for adding a `calcite-button` to the footer.
```

Comment:
should be `buttons`, they can have multiple

Reply:
🤙🏼